lib/active_admin_import/model.rb

```diff
@@ -40,7 +40,7 @@ def read_attribute_for_validation(key)
     end
 
     def default_attributes
-      {hint: '', file: nil, csv_headers: [], allow_archive: true, force_encoding: 'UTF-8'}
+      { hint: '', file: nil, csv_headers: [], allow_archive: true, force_encoding: 'UTF-8' }
```

Comment:
Line is too long. [92/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.